SCClassLibrary/JITLib/ProxySpace/BusPlug.sc

```diff
@@ -256,6 +256,7 @@ BusPlug : AbstractFunction {
 			^this
 		};
 		if(bus.rate == \control) { "Can't monitor a control rate bus.".warn; monitor.stop; ^this };
+		group = group ?? {this.homeServer.defaultGroup};
```

Comment:
so if i'm understanding correctly, you should update the usage of ? below as well then